_posts/2020-09-14-screenshots.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: newsletter
+title: "Newsletter #10 - What visual can I expect ?
+category: newsletter
+permalink: /news/2020/09/
+---
+
+What is Neovim?
+---------------
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html).
+
+### Participating
+
+If you are interested in contributing to Neovim, visit [the github
+page](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md) for details.
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](https://gitter.im/neovim) (also
+accessible via [IRC](https://webchat.freenode.net/?channels=neovim)), or visit
+the [Neovim subreddit](https://www.reddit.com/r/neovim/), which many of the
+developers read. Taking part in conversations is a contribution, too!
+
+
+Neovim: the beauty and the beast
+----------
+
+TUI = beast
+
+then move on to 
+
+![Courtesy of sunjon](../images/showcase/telescope_helptags.png)
+![Courtesy of smolck veonim](../images/showcase/veonim_nyancat.png)
```

Comment:
Please make sure to credit https://github.com/veonim/veonim for this picture ("official" pic: https://github.com/veonim/veonim/blob/release/docs/nyan.png)